io/plugins/eu.esdihumboldt.hale.io.csv.ui/src/eu/esdihumboldt/hale/io/csv/ui/TypeSelectionPage.java

```diff
@@ -122,16 +129,36 @@ public void selectionChanged(SelectionChangedEvent event) {
 		skipNlinesSpinner.setIncrement(1);
 		skipNlinesSpinner.setPageIncrement(10);
 
+//		create date formatter combo
+		Label dateFormatterLabel = new Label(page, SWT.NONE);
+		dateFormatterLabel.setText("Date formatter:");
```

Comment:
I'd suggest to put a more descriptive label here, e.g. `Format for imported date values`.